test/memote.jl

```diff
@@ -78,6 +78,23 @@ end
     wrong_model.metabolites["pyr_c"].formula = "C2H3X"
     @test !isempty(Consistency.reactions_charge_unbalanced(wrong_model))
     @test !isempty(Consistency.reactions_mass_unbalanced(wrong_model))
+
+    # test metabolite connectivity
+    dm = find_disconnected_metabolites(model)
```

Comment:
```suggestion
    dm = Consistency.find_disconnected_metabolites(model)
```
(these are a guess but it looks like the other functions do the same :D )